tuf/signed/verify_test.go

```diff
@@ -152,6 +163,10 @@ func TestUnknownKeyBelowThreshold(t *testing.T) {
 	s.Signatures = append(s.Signatures)
 	err = VerifySignatures(s, roleWithKeys)
 	require.IsType(t, ErrRoleThreshold{}, err)
+	require.Len(t, s.Signatures, 2)
+	// order of the list is sometimes flaky, so using contains instead
+	require.Contains(t, []bool{s.Signatures[0].IsValid, s.Signatures[1].IsValid}, true)
+	require.Contains(t, []bool{s.Signatures[0].IsValid, s.Signatures[1].IsValid}, false)
```

Comment:
nit: instead of doing it like this, could we have individual checks against each key as specified by key ID?  It'll be more verbose, but I think it'll be clearer to understand (and debug if the tests breaks)